src/content/protocols/pancakeswap-v2-bsc.md

```diff
@@ -0,0 +1,150 @@
+
+> Autonomy score: **Medium**
+
+## Exit Window
+
+Any update to farming parameters, whether adding new Farms, changing allocation points, or adjusting emission splits—must be queued through the protocol’s timelock and cannot execute until at least six hours after being scheduled. This enforced delay gives stakers a clear window to withdraw or migrate their tokens before the change takes effect.
+
+In contrast, governance can reassign the 0.08 % protocol fee at any time by calling the Factory’s fee-routing function. Because this action bypasses the timelock, fee redirection occurs instantly and leaves no exit window for LPs to react.
+
+Once a liquidity pair is deployed, its internal fee-distribution logic is permanent: the 0.17 % trading fee directed to liquidity providers cannot be turned off or paused by any admin or multisig. Moreover, neither the Pair nor MasterChef contracts include a freeze switch, so LPs always retain the ability to remove their liquidity on demand.
+
+> Exit-Window score: **High**
+
+## Accessibility
+
+Users primarily interact with PancakeSwap v2 through its official web interface at https://pancakeswap.finance, which is developed and maintained by Pancake Labs. For those who prefer a self-hosted setup—or wish to run a custom UI—PancakeSwap’s frontend code is published under an MIT license at https://github.com/pancakeswap/pancake-frontend. The repository’s README walks through cloning the project, installing dependencies, configuring environment variables, making it straightforward for anyone to deploy their own instance of the interface.
+
+> Accessibility score: **Medium**
+
+## Conclusion
+
+The PancakeSwap-v2 protocol receives **Low, Medium, Medium, High and Medium** risk scores for the Chain, Upgradeability, Autonomy, Exit-Window and Accessibility dimensions, respectively, placing it in **Stage 0**. This rating comes from the concentration of decisive authority in pairs of 3-of-6 multisigs and from Pancake Labs’ position as the sole distributor of the main web front-end, leaving users dependent on a single operator for both governance and interface.
+
+The protocol could reach Stage 1 by; bringing all fee-switch and CAKE-mint authorities—including in MasterChef V1 under a ≥ 7-signer, ≥ 51 %-quorum Security Council, and 2 guaranteeing that the same council or timelock gates every future change to fees and ownership.
+
+The project could advance to Stage 2 if every critical permission concerning CAKE fee incentive distribution was irrevocably assigned to on-chain governance and protected by a long Exit Window, with multiple independent front-ends ensuring continuous user access.
```

Comment:
```suggestion
The project could advance to Stage 2 if all critical permissions concerning `CAKE` incentive distribution were irrevocably assigned to on-chain governance and protected by a _30-day Exit Window_. In addition to those, it would require multiple independent front-ends ensuring continuous user access.
```